stage1/enter_kvm/enter_kvm.go

```diff
@@ -223,17 +231,22 @@ func execSSH() error {
 
 	// this should not return in case of success
 	err = syscall.Exec(sshPath, args, os.Environ())
-	return fmt.Errorf("cannot exec to ssh: %v", err)
+	return errwrap.Wrap(errors.New("cannot exec to ssh"), err)
 }
 
 func main() {
 	flag.Parse()
+
+	log = rktlog.New(os.Stderr, "enter_kvm", debug)
```

Comment:
I don't think this is not necessary - the logger is created in the `init` function above. I'm leaving it to you, whether the prefix should be "enter_kvm" or "stage1 kvm".